rewriter/Command.cc

```diff
@@ -84,9 +86,24 @@ void Command::run(core::MutableContext ctx, ast::ClassDef *klass) {
     ast::MethodDef::Flags flags;
     flags.isSelfMethod = true;
     flags.discardDef = true;
-    auto selfCall = ast::MK::SyntheticMethod(call->loc, call->loc, call->name, std::move(newArgs),
+    auto selfCall = ast::MK::SyntheticMethod(call->loc, call->declLoc, call->name, std::move(newArgs),
```

Comment:
Was this just an unrelated bug or was this directly implicated in this change?

Reply:
This was directly implicated in the change; the tests did not work without this change.